### PR TITLE
UI for personal times

### PR DESCRIPTION
**Pull request by louietyj**
Blocked on email reminders service

### CODE REVIEW

File: config/locales/en/course/personal_times.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    personal_times:
+      index:
+        header: 'Personalized Timelines'
+        assessment: 'Assessment'
+        reference_timeline: 'Reference Timeline'
+        personalized_timeline: 'Personalized Timeline'
+        start_at: :'course.assessment.assessments.index.start_at'
+        bonus_end_at: :'course.assessment.assessments.index.bonus_cut_off'
+        end_at: :'course.assessment.assessments.index.end_at'
+        fixed_desc: 'A fixed personal time means that the personal time will no longer be automatically modified. If a
```
Comment (jeremyyap) [inline config/locales/en/course/personal_times.yml]: Use > for multiline strings

File: app/views/course/lesson_plan/items/_personal_or_ref_time.html.slim [hunk 2]
```diff
@@ -0,0 +1,10 @@
+- if item.time_for(course_user).is_a? Course::PersonalTime and item.time_for(course_user).fixed?
+  span title=t('course.lesson_plan.items.fixed_desc')
+    = fa_icon 'lock'
+  = ' '
```
Comment (jeremyyap) [inline app/views/course/lesson_plan/items/_personal_or_ref_time.html.slim]: Can do `=> fa_icon 'lock'` which will insert the space after the icon

File: app/views/course/lesson_plan/items/_personal_or_ref_time.html.slim [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+= format_datetime(item.time_for(course_user)[attrib], datetime_format)
```
Comment (jeremyyap) [inline app/views/course/lesson_plan/items/_personal_or_ref_time.html.slim]: can spell out attribute in full?

**Comment by louietyj** [comment]
🤔 I could make a course come with the boolean set to true so that my code cov goes up...

**Pull request merged**